[2024-03-03T03:42:24.658+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_2 scheduled__2017-01-02T00:00:00+00:00 [queued]>
[2024-03-03T03:42:24.670+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_2 scheduled__2017-01-02T00:00:00+00:00 [queued]>
[2024-03-03T03:42:24.671+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:42:24.694+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_2> on 2017-01-02 00:00:00+00:00
[2024-03-03T03:42:24.702+0000] {standard_task_runner.py:60} INFO - Started process 4850 to run task
[2024-03-03T03:42:24.707+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_2', 'scheduled__2017-01-02T00:00:00+00:00', '--job-id', '2219', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmpu7qwa0au']
[2024-03-03T03:42:24.710+0000] {standard_task_runner.py:88} INFO - Job 2219: Subtask get_files_2
[2024-03-03T03:42:24.742+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:42:24.814+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_2 scheduled__2017-01-02T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:42:24.940+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_2' AIRFLOW_CTX_EXECUTION_DATE='2017-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2017-01-02T00:00:00+00:00'
[2024-03-03T03:42:27.638+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:42:27.659+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_2, execution_date=20170102T000000, start_date=20240303T034224, end_date=20240303T034227
[2024-03-03T03:42:27.698+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:42:27.728+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
